tests/extmod/vfs_posix_nonwindows.py

```diff
@@ -0,0 +1,32 @@
+# Test for VfsPosix error conditions that cannot be reproduced on Windows
+
+try:
+    import os
+    import sys
+
+    os.VfsPosix
+except (ImportError, AttributeError):
+    print("SKIP")
+    raise SystemExit
+
+if sys.platform == "win32":
+    # Windows doesn't let delete the current directory, so this cannot be
+    # tested.
+    print("SKIP")
+    raise SystemExit
+
+curdir = os.getcwd()
+os.mkdir("aaa")
```

Comment:
I suggest to use a more unique name, eg "vfs_posix_test_dir".

Reply:
Done. I chose “vfs_posix_enoent_test_dir” in analogy with the vfs_posix_ilistdir_* tests, and also added the skip-if-already-exists from there.